DocumentResolver/DocumentResolver.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+namespace Simgroep\ConcurrentSpiderBundle\DocumentResolver;
+
+use VDB\Spider\Resource;
+use Simgroep\ConcurrentSpiderBundle\DocumentResolver\Type\Html;
+use Simgroep\ConcurrentSpiderBundle\DocumentResolver\Type\Pdf;
+use Simgroep\ConcurrentSpiderBundle\DocumentResolver\Type\MsDoc as MsDocType;
+use Simgroep\ConcurrentSpiderBundle\DocumentResolver\Type\Word2007;
+use Simgroep\ConcurrentSpiderBundle\DocumentResolver\Type\Rtf;
+use Simgroep\ConcurrentSpiderBundle\DocumentResolver\Type\Odt;
+
+/**
+ * Determine and extract document content from resource
+ */
+class DocumentResolver
+{
+    /**
+     * @var \Simgroep\ConcurrentSpiderBundle\DocumentResolver\Type\Html
+     */
+    private $html;
+
+    /**
+     * @var \Simgroep\ConcurrentSpiderBundle\DocumentResolver\Type\Pdf
+     */
+    private $pdf;
+
+    /**
+     * @var \Simgroep\ConcurrentSpiderBundle\DocumentResolver\Type\MsDoc
+     */
+    private $msdoc;
+
+    /**
+     * @var \Simgroep\ConcurrentSpiderBundle\DocumentResolver\Type\Word2007
+     */
+    private $word2007;
+
+    /**
+     * @var \Simgroep\ConcurrentSpiderBundle\DocumentResolver\Type\Rtf
+     */
+    private $rtf;
+
+    /**
+     * @var \Simgroep\ConcurrentSpiderBundle\DocumentResolver\Type\Odt
+     */
+    private $odt;
+
+    /**
+     * @var string
+     */
+    private $data;
+
+    /**
+     * Cosntructor
+     *
+     * @param Html $html
```

Comment:
In the rest of the codebase we use full namespaces everywhere. I think it's more pretty to keep it the same.